keyboards/nyquist/rev2/rev2.c

```diff
@@ -0,0 +1,39 @@
+#include "rev2.h"
+
+#ifdef AUDIO_ENABLE
+    float tone_startup[][2] = SONG(STARTUP_SOUND);
+    float tone_goodbye[][2] = SONG(GOODBYE_SOUND);
+#endif
+
+#ifdef SSD1306OLED
+void led_set_kb(uint8_t usb_led) {
+    // put your keyboard LED indicator (ex: Caps Lock LED) toggling code here
+    led_set_user(usb_led);
+}
+#endif
+
+void matrix_init_kb(void) {
+
+    #ifdef AUDIO_ENABLE
```

Comment:
This section will break startup sounds.  They're called elsewhere, and override the behavior, removing the option for user configuration (eg config.h override)